internal/handlers/common/update.go

```diff
@@ -236,7 +238,7 @@ func processPopFieldExpression(doc *types.Document, update *types.Document) (boo
 
 		err = doc.SetByPath(path, array)
 		if err != nil {
-			return false, err
+			return false, lazyerrors.Error(err)
```

Comment:
There is no test case for it, and I couldn't come up with one quickly, so I return `lazyerrors.Error` to make the fact we don't test it explicit.